recipes/scicookie/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "scicookie" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scicookie-{{ version }}.tar.gz
+  sha256: e17a7641a8c0ed6bc27e7826a0db45db4d6aa9dfc5e939ca7718df6ffb5030ac
+
+build:
+  skip: true  # [py>=400 or py2k or win]
```

Looks like this one can be noarch. BTW, grayskull added pybind11 and compilers here. That is odd b/c those are not in the pyproject.toml. It may be worth opening an issue upstream in grayskull.

```suggestion
  noarch: python
```